Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,10 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Like most applications, the easiest way to run rkt is to install it with your system's package manager, like apt on Debian or dnf on Fedora. If your distribution doesn't include a rkt package, or your operating system isn't Linux at all, running rkt in a Vagrant virtual machine can be nearly as simple. The instructions below start a virtual machine with rkt installed and ready to run.
```

Comment:
These are separate use cases. There are three use cases:

1) I want the latest rkt on my machines: download from the GitHub tarballs
2) I want my official package, install from apt or dnf
3) I don't have a linux machine, use vagrant.